Themes/default/scripts/jquery.sceditor.smf.js

```diff
@@ -767,13 +767,11 @@ sceditor.formats.bbcode.set(
 	'url', {
 		allowsEmpty: true,
 		quoteType: sceditor.BBCodeParser.QuoteType.always,
-		tags: {
-			a: {
-				'data-type': ['url']
-			}
-		},
-		format: function (element, content)
+		format(element, content)
```

Comment:
Why this change? It works, but it is different than anything we do elsewhere. Should we make similar changes elsewhere in this file?

Reply:
This is ES6 syntax and is a copy from my mod. I can revert it.